### PR TITLE
feat(locale): add location for uz_UZ_latin

### PR DESCRIPTION
**Pull request by Mirazyzz**

### CODE REVIEW

File: src/locales/uz_UZ_latin/location/street_pattern.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default [
+  '{{location.street_suffix}} {{location.street_name}}',
```
Comment (ST-DDT) [inline src/locales/uz_UZ_latin/location/street_pattern.ts]: This looks strange to me.
Why does it start with the suffix?
Also IMO we should name the street_name file street_name_part or similar.

**Review by ST-DDT — COMMENTED** [review]
Please sort the files alphabetically.

File: src/locales/uz_UZ_latin/location/street_pattern.ts (hunk 1 above)
Comment (Mirazyzz) [reply src/locales/uz_UZ_latin/location/street_pattern.ts]: I missed that one, suffix should be after the street name.

**Comment by Mirazyzz** [comment]
> Please sort the files alphabetically.

Do you want the street names to be sorted alphabetically? They're pretty large and looking at other locales I thought it's not necessary.

**Comment by Shinigami92** [comment]
I suggest to add the locale to the PR title

**Review by Shinigami92 — DISMISSED** [review]
Count me already as approved but fix the change request from @ST-DDT pls

**Comment by matthewmayer** [comment]
> > Please sort the files alphabetically.
> 
> 
> 
> Do you want the street names to be sorted alphabetically? They're pretty large and looking at other locales I thought it's not necessary.

All *new* locale data should be sorted alphabetically unless there's a good reason (eg months of the year can be kept as Jan-Feb-mar...)

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by Mirazyzz** [comment]
> Please make the requested changes.

Ready! sorry for the delay.

File: src/locales/uz_UZ_latin/location/street_pattern.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export default ['{{location.street_name}} {{location.street_suffix}}'];
```
Comment (matthewmayer) [inline src/locales/uz_UZ_latin/location/street_pattern.ts]: This should be changed to location.street_name_part now right?

**Review by matthewmayer — CHANGES_REQUESTED** [review]

File: src/locales/uz_UZ_latin/location/street_pattern.ts (hunk 2 above)
Comment (Mirazyzz) [reply src/locales/uz_UZ_latin/location/street_pattern.ts]: exactly, my bad.

**Review by ST-DDT — DISMISSED** [review]

**Review by Shinigami92 — DISMISSED** [review]

File: src/locales/uz_UZ_latin/location/street_address.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export default {
+  normal: '{{location.street}}, {{location.buildingNumber}}',
+  full: '{{location.street}}, {{location.buildingNumber}} {{location.secondaryAddress}}',
```
Comment (matthewmayer) [inline src/locales/uz_UZ_latin/location/street_address.ts]: Is this correct?
If a secondary address (e.g. room/apartment number is provided) this immediately follows the building number with no comma etc?

without secondary address:
```
fakerUZ_UZ_latin.location.streetAddress()
> Munif Mahanov qishlog'i, 905
> Sobit Miranov xiyoboni, 35a
```

with secondary address:
```
fakerUZ_UZ_latin.location.streetAddress(true)
> Munif Mahanov qishlog'i, 905 972 uy
> Sobit Miranov xiyoboni, 35a 817 xonadon
```

**Review by matthewmayer — CHANGES_REQUESTED** [review]

File: src/locales/uz_UZ_latin/location/street_address.ts (hunk 3 above)
Comment (Mirazyzz) [reply src/locales/uz_UZ_latin/location/street_address.ts]: Yes, you're right I think it's better to separate them with comma. By the way, it's also better to put "-" for secondary address between house number and title (uy, xonadon). In Uzbek "-" is used for ordinal numbers. Do you think I should change it too?

Comment (matthewmayer) [reply src/locales/uz_UZ_latin/location/street_address.ts]: Yes

**Review by matthewmayer — APPROVED** [review]

**Review by Shinigami92 — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

**Pull request merged**